### DISCUSSION TITLE
When are translations merged to production?

### DISCUSSION THREAD

**Discussion by bmaster001** [pretix/pretix#3811] [Q&A] [answered]
I'm currently trying out pretix with the Dutch translation, and noticed that one string is still showing up in English, while there is a Dutch translation already done 2 months ago (https://translate.pretix.eu/translate/pretix/pretix/nl/?q=After+completing+your+purchase%2C+we+will+ask+you+to+transfer+the+money+to+our+bank+account%2C+using+a+personal+reference+code.&sort_by=-priority%2Cposition&offset=1). 
When are translations usually merged into production?

**Comment by raphaelm** [comment]
Every few days, so after 2 months there must be something different wrong. can you send a screenshot where exactly it is wrong?

**Comment by bmaster001** [comment]
Yes, of course. See below, under the blue test-message:
![image](https://github.com/pretix/pretix/assets/6114149/eb5ceab5-5cac-4de5-8151-d30242d0f9b6)

**Reply by raphaelm** [reply, marked as answer]
I've just seen, the translation is not 2 months old, it is 14 hours old, so it will be in production later this week. The one from 2 months ago is not used because it has the "needs editing" flag:
![image](https://github.com/pretix/pretix/assets/64280/f4a5c34b-925c-4230-a80a-698edc37f88e)

**Reply by bmaster001** [reply]
Ooh, I see it now... Sorry, my bad! Thanks for the clarification :-)
